js/collapse.js

```diff
@@ -154,6 +149,21 @@
       .attr('aria-expanded', isOpen)
   }
 
+  Collapse.prototype.updateTriggers = function ($currentTrigger) {
+    this.$allTriggers  = $('[data-toggle="collapse"][href="#' + this.$element[0].id + '"],'
```

Comment:
Should these also use `options.trigger` instead of hardcoding `[data-toggle="collapse"]` ?